ProjectTemplates/DotNetTemplate/MonoGame.Templates.CSharp/content/MonoGame.Application.UWP.CoreApp.CSharp/project.json

```diff
@@ -0,0 +1,16 @@
+﻿{
+  "dependencies": {
+    "Microsoft.NETCore.UniversalWindowsPlatform": "5.2.2"
```

Comment:
I think this file can be removed, because the csproj has a PackageReference set. The Include for this file in the csproj should also be removed.

Reply:
Good catch!